rotkehlchen/chain/ethereum/modules/l2/loopring.py

```diff
@@ -252,7 +253,10 @@ def _api_query(
                 result_info = json_ret.get('resultInfo', None)
                 if result_info:
                     code = result_info.get('code', None)
-                    if code and code == 104002:
+                    # This two codes can appear if there is an error with the
+                    # key as tested here
+                    # https://github.com/rotki/rotki/pull/2569#issuecomment-800151821
+                    if code and code in [100001, 101002]:
```

Comment:
super nitpick: In such cases don't use list. use a tuple. Since it's immutable and better optimized
```suggestion
                    if code and code in (100001, 101002):
```